backend/src/agents/intent_agent.py

```diff
@@ -18,7 +18,6 @@
     tools=[],
 )
 class IntentAgent(Agent):
-
     async def invoke(self, utterance: str) -> str:
         user_prompt = engine.load_prompt("intent", question=utterance, chat_history=get_session_chat())

```

Comment:
```suggestion
        session_chat=get_session_chat()
        user_prompt = engine.load_prompt("intent", question=utterance, chat_history=session_chat if session_chat else "There is no chat history")

```
Would this improve the prompt?